build.sh

```diff
@@ -25,10 +25,10 @@ fi
 echo "Working directory is: $(pwd)"
 
 # We prefer dotnet if available as the package size is (much) smaller.
-if which dotnet > /dev/null 2>&1 && [ "$(dotnet --version | cut -f1 -d.)" == "5" ]; then
+if which dotnet > /dev/null 2>&1 && [ "$(dotnet --version | cut -f1 -d.)" -ge 5 ]; then
```

Comment:
This is good enough for compatibility and future proof.

Ideally, it should match the installed/chosen .NET version if setup-dotnet step is run... But I can't think of a reliable way to check that without adding extra clobber for the end user; so `-ge 5` should be good enough. Perhaps limit to the supported version numbers (check `-ge 5` **and** `-le 7`). But then there should be a reliable way to update this check every time .NET version support is upgraded.